src/tags/apply.js

```diff
@@ -12,19 +12,28 @@ const Builder = require('../structures/TagBuilder'),
 
 module.exports =
     Builder.ArrayTag('apply')
-        .withArgs(a => [a.require('subtag'), a.optional('args', true)])
-        .withDesc('Executes `subtag`, using the `args` as parameters. ' +
-        'If `args` is an array, it will get deconstructed to it\'s individual elements.'
+        .withArgs(a => [a.require([a.optional('subtag'), a.optional('function')]), a.optional('args', true)])
+        .withDesc('Executes `subtag` or `function`, using the `args` as parameters. ' +
+            'If `args` is an array, it will get deconstructed to it\'s individual elements.\n' +
+            '`function` must be of the format `func.<name>` eg: `{apply;func.hello;["world"]}`'
         ).withExample(
         '{apply;randint;[1,4]}',
         '3'
         )
         .whenArgs(0, Builder.errors.notEnoughArguments)
         .whenDefault(async function (subtag, context, args) {
-            let definition = TagManager.get(args[0].toLowerCase());
-            if (definition == null)
+            let runSubtag;
+            const name = args[0].toLowerCase();
+            if (context.state.overrides.hasOwnProperty(name)) {
+                runSubtag = context.state.overrides[name];
+            } else {
+                const tagDefinition = TagManager.get(name) || {};
+                runSubtag = context.state.overrides[tagDefinition.name] || tagDefinition.execute;
+            }
+            if (runSubtag === undefined)
```

Comment:
Change this to a nullish check, just to be safe

Reply:
Yeah I looked at the code a bit and it *should* only be `undefined` or the actual override, but I guess it won't hurt to be safe.